### DISCUSSION TITLE
Should I make a ToonSquid Website or App?

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#648] [Polls]
Hey guys, so I was thinking of making an app or website that would allow people in ToonSquid to trade brushes/assets, write tutorials, show animations (maybe), and more (these are just the ideas I got).  The only problem I have is that I don't know if it should be a website or an app. I asked the discord and now what to ask here. What do you think it should be?

**Comment by AFC26** [comment]
I like the idea if it’s for the progress of the ToonSquid app and the users of the app.

**Reply by budaloops** [reply]
It’s a good idea. Isn’t Discord better for this? There are places like Reddit and Discord that people already use as artist communities. It may be hard at first to get them to flock over to your app, which is kind of out of the way because it has to be downloaded. But who knows? If it excites you then you’ll be able to take it far

**Reply by Gottafindaname** [reply]
I agree discord is better for that…. We share all that in the one im in
